external/microbit_v2/Board.mk

```diff
@@ -60,7 +60,6 @@ BOARD_SOURCES += \
 	nrf_strerror.c\
 	nrf_twi_mngr.c\
 	nrfx_clock.c\
-	nrfx_gpiote.c\
```

Comment:
nrfx gpiote implements its own handler, which causes linker errors. We don't want nrfx anyway (thats why we have edge::aidan)

Reply:
nice